### PR TITLE
[MIRROR] Reverts recipe changes from Industrial Expansion

### PR DESCRIPTION
**Pull request by VirgoBot**
Original PR: https://github.com/PolarisSS13/Polaris/pull/8075
--------------------
Blanket revert of recipe changes by #7811 
Part 2 of, and afaict fixes #7885

These changes weren't sufficiently discussed, and have been met with widespread disapproval at the increased complexity, tedium, and reliance upon mining. Sufficient quantities of copper, titanium, and aluminum were not mapped in to accompany these changes, meaning cargo, engineering, and research are far more reliant upon mining to do various pieces of their jobs.

Recipes may later be changed to use these new materials, but not before more serious consideration of how accessible the materials for those recipes should be.

### CODE REVIEW

**Comment by Heroman3003** [comment]
#9904 will have to be reverted as well.

File: code/modules/research/designs/circuits/circuits.dm [hunk 1]
```diff
@@ -662,19 +662,6 @@ CIRCUITS BELOW
...
-	req_tech = list(TECH_DATA = 2, TECH_ENGINEERING = 3, TECH_COMBAT = 4)
-	build_path = /obj/item/weapon/circuitboard/pointdefense
-	sort_string = "OAABA"
-
-/datum/design/circuit/pointdefense_control
-	name = "point defense control"
-	id = "pointdefense_control"
-	req_tech = list(TECH_DATA = 4, TECH_ENGINEERING = 3, TECH_COMBAT = 2)
-	build_path = /obj/item/weapon/circuitboard/pointdefense_control
-	sort_string = "OAABB"
```
Comment (TheFurryFeline) [inline code/modules/research/designs/circuits/circuits.dm:677]: Do we want this removed?

Comment (Novacat) [reply code/modules/research/designs/circuits/circuits.dm:677]: Probably not.

**Comment by TheFurryFeline** [comment]
Alright, then

**Pull request merged**